modules/tensor_mechanics/src/materials/ComputeIsotropicElasticityTensor.C

```diff
@@ -63,56 +65,103 @@ ComputeIsotropicElasticityTensor::ComputeIsotropicElasticityTensor(
   if (_youngs_modulus_set && _youngs_modulus <= 0.0)
     mooseError("Youngs modulus must be positive in material '" + name() + "'.");
 
+  std::vector<Real> iso_const(2);
```

Comment:
Optimization suggestion:
```
std::array<Real, 2> iso_const;
```
This will create stack allocated storage versus heap allocated storage, which we've found does slow down code that is called repeatedly like this routine.

Reply:
The method _C_ijkl.fillFromInputVector requires vector<Real> as an input, which is iso_const . Should I define iso_const as an array and then convert it to vector real? Or should I let it be?

Reply:
Leave it - this is just an optimization suggestion. In the future, somebody might overload the method in the tensor class so that it can accept either type. Converting is expensive too so there's no reason to change types now.